content/telegraf/v1.18/about_the_project/release-notes-changelog.md

```diff
@@ -7,6 +7,45 @@ menu:
     weight: 10
     parent: About the project
 ---
+
+## v1.18.1 [2021-04-07]
+
+- Agent: Closes running outputs when agent reloads on SIGHUP.
+
+### Input plugin updates
+
+- [Docker Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/docker)(`docker`):
+  Fix panic when parsing container statistics.
+- [Exec Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/exec)(`exec`):
+  Fix truncated messages in debug mode; debug mode now shows full messages.
+- [IPMI Sensor Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/ipmi_sensor)(`ipmi_sensor`):
+  Fix panic by implementing a length check to plugin.
+- [MySQL Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mysql)(`mysql`):
+  Fix the ability to handle ‘binary logs’ query for MySQL version 8.0+.
+- [NFS Client Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/nfsclient)(`nfsclient`):
+  Fix integer overflow in fields received by mountstat.
+- [Ping Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/ping)(`ping`):
+  Fix an issue when `method=”native”` by always calling `SetPrivileged(true)`.
```

Comment:
I'm not sure what makes sense to say here? Maybe:
Resolve error that prevented the agent from running when an unprivileged UDP ping was sent. Now, `SetPrivileged(true)` is always true in native mode to ensure a privileged ICMP ping is sent.